pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -308,8 +309,7 @@ func (p *passThroughClient) Request(ctx context.Context, request *networkservice
 		return nil, err
 	}
 
-	request.Connection.Path.Index += conn.Path.Index
-	request.Connection.Path.PathSegments = append(request.Connection.Path.PathSegments, conn.Path.PathSegments...)
+	request.Connection.Path.PathSegments = conn.Path.PathSegments
```

Comment:
Do we need to move index here?

Reply:
No, because index should point at the current path segment.